src/Commands/TankDrive.cpp

```diff
@@ -2,10 +2,8 @@
 #include "TankDrive.h"
 #include "../Subsystems/Chassis.h"
 
-using namespace robovikes;
-
 TankDrive::TankDrive(bool automaticShifting) :
-	mAutomaticShifting(automaticShifting) {
+	mAutomaticShifting(true) {
```

Comment:
Why does `TankDrive` have a parameter that you are ignoring? 

The original line was correct. Remove this line.